tests/test_cli.py

```diff
@@ -140,3 +142,47 @@ def test_inject_column_sort(package_path, column_sort_arg):
 def test_version():
     result = runner.invoke(app, ["version"])
     assert result.exit_code == 0
+
+from click.testing import CliRunner
+from paracelsus.cli import cli
+
+
+
+def test_graph_with_inclusion_regex(package_path: Path):
+    result = runner.invoke(
+        app,
+        [
+            "graph",
+            "example.base:Base",
+            "--import-module",
+            "example.models",
+            "--python-dir",
+            str(package_path),
+            "--include-tables",
+            "^com.*",
+        ],
+    )
+    assert result.exit_code == 0
+    assert "regular_table {" not in result.stdout
+    assert "first_test {" in result.stdout
+    assert "second{" not in result.stdout
```

Comment:
These tests don't make sense- I don't know where these strings are coming from. The goal here is to test that the proper columns exist, but the excluded ones do not.

In this case we have three tables- `comments`, `users`, and `posts`. Since we're trying to include the pattern `^com.*` we expect the comments table to be in the diagram but not the other two.

```python
assert "comments {" in result.stdout
assert "users {" not in result.stdout
assert "post {" not in result.stdout
```

Reply:
I apologise @tedivm I'll make changes !

Reply:
I apologise @tedivm I'll make changes !

Reply:
No worries! I appreciate you taking this on!

Reply:
I apologise @tedivm I'll make changes !